src/clients/QBittorrent.ts

```diff
@@ -425,7 +425,7 @@ export default class QBittorrent implements TorrentClient {
 						category: searcheeInfo!.category,
 					};
 			if (!isComplete) return InjectionResult.TORRENT_NOT_COMPLETE;
-			const filename = `${newTorrent.getFileSystemSafeName()}.${TORRENT_TAG}.torrent`;
+			const filename = `${newTorrent.getFileSystemSafeName()}.${crossSeedTag}.torrent`;
```

Comment:
This replacement, while somewhat consistent, is not really required. I can't see where it would make a difference unless you save the torrent file added to qbittorrent on add/completion in the client - which it would then be reflected in the file saved to your specified directory this way. 

I think leaving this as TORRENT_TAG would make a clear distinction that would not generally affect the workflow. 

Just my opinion.